### PR TITLE
Refactor menubar.js with Aphrodite

### LINKED ISSUE #7372: Refactor menubar with Aphrodite
## Test plan

https://github.com/brave/browser-laptop/pull/11587#issue-266365280

----

Refactor menubar.js with Aphrodite

### PR DESCRIPTION
**Pull request by luixxiul**
Fixes #7372

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [ ] Ran `git rebase -i` to squash commits (if needed).

Test Plan: TBD

### CODE REVIEW

**Comment by luixxiul** [comment]
@cezaraugusto I'll squash the commits as usual.

**Comment by cezaraugusto** [comment]
@luixxiul is this ready for review? Test plan is written as TBD

**Comment by luixxiul** [comment]
I will update it later.

File: app/renderer/components/windowCaptionButtons.js [hunk 1]
```diff
@@ -55,7 +55,8 @@ class WindowCaptionButtons extends ImmutableComponent {
       fullscreen: this.props.windowMaximized,
       windowCaptionButtons: true,
       verticallyCenter: this.props.verticallyCenter
-    })}>
+    })}
+      style={{flexGlow: 0}}> // TODO: Remove this styling by refactoring windowCaptionButtons.js
```
Comment (bsclifton) [inline app/renderer/components/windowCaptionButtons.js]: also: is it supposed to be flexGrow?

Comment (luixxiul) [inline app/renderer/components/windowCaptionButtons.js]: based on https://github.com/brave/browser-laptop/pull/7373/files#diff-02c4b23ad267fe636760179e32fa29ceL437, I think so.

**Comment by luixxiul** [comment]
@bsclifton 
> it may be using selectors (which we'd need to update)

would you let me know which ones I should update; otherwise could you submit a fix to this PR?

**Comment by bsclifton** [comment]
moving to 0.13.7

**Comment by cezaraugusto** [comment]
I can help here but not quite sure what's missing?

**Comment by bsclifton** [comment]
@cezaraugusto I believe the arrow keys issue may have been resolved with inclusion of LocalShortcuts (which I fixed with https://github.com/brave/browser-laptop/commit/5a889ccd6198906537f0c29d4b28067292a4ef78)

We'll want to:
- remove the comment which is breaking the UI (as shown above)
- fix naming FlexGlow => FlexGrow (also above)

Then we can make sure webdriver tests pass OK and I think we're good to go 😄

**Comment by luixxiul** [comment]
> fix naming FlexGlow => FlexGrow (also above)

Yes right! Sorry for my poor English :-p

**Comment by bsclifton** [comment]
Closing this PR as stale.  If you revisit and implement the feedback, please feel free to reopen / submit new PR. Thanks 😄

**Pull request closed without merging**